src/rendering/contexts/native/invoker.zig

```diff
@@ -350,13 +355,14 @@ pub fn InvokerType(
         }
 
         pub inline fn get(
+            data_render: *DataRender,
             data: anytype,
             path: Element.Path,
             index: ?usize,
         ) PathResolutionType(Context) {
             const GetPathInvoker = PathInvokerType(error{}, Context, getAction);
             return GetPathInvoker.call(
-                {},
+                .{data_render},
```

Comment:
Here a first shortcut I made:
Now the `action_param` parameter from the `call` function is always a tuple with a `*DataRender` object as first parameter (for the 4 functions where it is used (`get`, `interpolate`, `capacityHint` and `expandLambda`).
Maybe we could make this `*DataRender` a new concrete parameter for the `call` function ?

Reply:
Fixed by: https://github.com/batiati/mustache-zig/pull/24/commits/38994437873cf1cfc067b6c70194444fbbc53b35